tests/test_solver_differences.py

```diff
@@ -144,8 +144,17 @@ def test_gpu_cpu_mutexes():
         env=env,
     )
     data = json.loads(p.stdout)
-    # This should not happen, but it does. See docstring.
-    assert next(pkg for pkg in data["actions"]["LINK"] if pkg["name"] == "cudatoolkit")
+    # After recent libsolv/libmamba updates, this now works correctly.
```

Comment:
Can you try to pinpoint in which version the fix was introduced? 🙏

Reply:
I had the same question (as I added in the PR description) – I'll try to scope out the difference with what I can see from CI runs.

Reply:
or "pytorch repackaging" etc?